cmd/kubeadm/app/master/manifests.go

```diff
@@ -233,7 +233,7 @@ func getComponentCommand(component string, s *kubeadmapi.MasterConfiguration) (c
 			"--tls-cert-file=" + pkiDir + "/apiserver.pem",
 			"--tls-private-key-file=" + pkiDir + "/apiserver-key.pem",
 			"--token-auth-file=" + pkiDir + "/tokens.csv",
-			"--secure-port=443",
+			fmt.Sprintf("--secure-port=%d", s.API.BindPort),
```

Comment:
I am renaming it throughout now, however please note that there is a one-letter variable convention throughout the codebase, it's used for structs like this, which are sort of the single most important thing in a given file... Not to say that we have to adhere to this convention, it's probably not the greatest one there is.